scripts/Tools/check_input_data

```diff
@@ -41,10 +38,11 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
     parser.add_argument("--svn-loc", default=SVN_LOCS[get_model()],
                         help="The input data repository from which to download data.")
 
-    parser.add_argument("-i", "--input-data-root", default=None,
-                        help="The root directory where input data goes. "
-                        "Default value will come from DIN_LOC_ROOT from case which itself "
-                        "defaults to '{}'".format(MACHINE.get_value("DIN_LOC_ROOT")))
+    din_loc_root = run_cmd_no_fail("./xmlquery --value DIN_LOC_ROOT")
```

Comment:
The only reason to have this there is to show the user the default value in the help message.
How about if instead the help says "Use xmlquery DIN_LOC_ROOT in your case directory to view default."